aptos-move/block-executor/src/task.rs

```diff
@@ -47,7 +47,19 @@ pub trait Transaction: Sync + Send + Clone + 'static {
         + DeserializeOwned
         + Serialize;
     /// AggregatorV2 identifier type.
-    type Identifier: PartialOrd + Ord + Send + Sync + Clone + Hash + Eq + Debug;
+    type Identifier: PartialOrd
+        + Ord
+        + Send
+        + Sync
+        + Copy
+        + Clone
+        + Hash
+        + Eq
+        + Debug
+        // Identifiers are generated from u32 and values are exchanged with u64.
+        + From<u64>
+        // Value exchange uses u64.
+        + Into<u64>;
```

Comment:
hm, if you are using this, how are you going to exchange AggregatorSnpshot<String> ?

Reply:
This is for identifiers? They are always u64, but because we use AggregatorID wrappers, we need to be able to get the inner value.

Reply:
I see, so you take a identifier, transform into u64, and then do to_string if it is AggregatorSnapshot<String>